bots/star-of-david.bot.js

```diff
@@ -0,0 +1,8 @@
+const Slack = require('pico-slack');
+
+Slack.onReact((reaction) => {
+	const isProbablyADave = reaction.reaction.indexOf('dav') >= 0;
```

Comment:
`reaction.indexOf('_dave') !== 0 || reaction.startsWith('dav')` Might be more accurate

Reply:
I originally had `indexOf('david') >= 0 || indexOf('dave') >= 0` but this is more concise and matches all the current Dave emojis. Also, it will be funny if we ever add `:cadaver:` or `:affidavit:`.

Reply:
Well dammit, I was looking forward to popping in with a comment saying "I can't wait to add a `:cadaver:` emoji," and here you already beat me to it! :persevere: